experiments/snippets/realTimeOppDashboard/RealTimeOpportunityDashboard.xml

```diff
@@ -0,0 +1,245 @@
+<skuidpage showsidebar="false" showheader="true" tabtooverride="Opportunity" unsavedchangeswarning="" theme="">
```

Comment:
```suggestion
<skuidpage showsidebar="false" showheader="true" tabtooverride="Opportunity" unsavedchangeswarning="" >
```

Leaving this empty theme attribute causes an error at runtime about the theme not being found. Removing it fixes the issue.

Reply:
Sounds good.  Fixed in latest commit.